src/Plugin/HelpSection/ProfileHelpMaintainingSection.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Drupal\stanford_profile\Plugin\HelpSection;
+
+use Drupal\help\Plugin\HelpSection\HelpSectionPluginBase;
+
+/**
+ * Provides the module topics list section for the help page.
+ *
+ * @HelpSection(
+ *   id = "profile_help_maintaining",
+ *   title = @Translation("Maintaining your site"),
+ *   description = @Translation("Resources for maintaining the health of your site."),
+ *   weight = -99
+ * )
+ */
+class ProfileHelpMaintainingSection extends HelpSectionPluginBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function listTopics() {
+    return [
+      $this->getMaintainingYourSite(),
+      $this->getStanfordWebServicesSupport(),
+      $this->getSiteHealth(),
+      $this->getContent(),
+    ];
+  }
+
+  /**
+   * Get the user guide help text.
+   *
+   * @return array
+   *   Markup render array.
+   */
+  protected function getMaintainingYourSite() {
+    $help = '<h3>' . $this->t('Maintaining your site') . '</h3>';
+    $help .= '<p>' . $this->t('The following tools and resources will help you keep your site healthy. You can also join our active community of Stanford Sites users on  <a href="https://stanford.enterprise.slack.com/archives/C01NR8WC6AX">Slack</a>. This is a great place to ask questions, learn about the newest features on Stanford Sites, and more!') . '</p>';
+    return ['#markup' => $help];
+  }
+
+  /**
+   * Get the user guide help text.
+   *
+   * @return array
+   *   Markup render array.
+   */
+  protected function getStanfordWebServicesSupport() {
+    $help = '<h3>' . $this->t('Stanford Web Services Support') . '</h3>';
+    $help .= '<p>' . $this->t('The following tools and resources will help you keep your site healthy. You can also join our active community of Stanford Sites users on Slack. This is a great place to ask questions, learn about the newest features on Stanford Sites, and more!') . '</p>';
```

Comment:
This should be the first section under "Maintaining Your Site" Let's change the text under  the "Support from Stanford Web Services" header to: 
Need expert advice or just an extra pair of hands? Stanford Web Services has you covered, with professional services to meet your needs. 

-[ Digital Strategy Consulting](https://uit.stanford.edu/service/digital-strategy)
- [Content Care Service](https://uit.stanford.edu/service/website-accessibility)
- [Website Support Retainer](https://uit.stanford.edu/service/website-support-retainer)
- And [more!](https://uit.stanford.edu/sws)